### PR TITLE
fix webpack silently failing; update package.json

### PR DESCRIPTION
**Pull request by mbacchi**
packages: typescript and awesome-typescript-loader versions

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -99,7 +99,7 @@
     "react-redux": "^5.0.6",
     "redux": "^3.7.2",
     "redux-logger": "^3.0.6",
-    "redux-thunk": "^2.2.0",
+    "redux-thunk": "2.2.0",
```
Comment (bridiver) [inline package.json]: interesting, I guess I must have had 2.2.0 so I didn't need this

**Comment by bbondy** [comment]
Merged and also updated the ref on brave-core here:
https://github.com/brave/brave-core/commit/278e71f883f3688f46122ce2ad1a78dfa4fc3769

**Comment by garrettr** [comment]
Shouldn't this PR have updated the version controlled `yarn.lock` in addition to `package.json`? 

Related question: should `package-lock.json` be version controlled in this repo?

**Comment by bbondy** [comment]
I think package-lock.json is meant to be in source control.

**Pull request merged**